Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```
@@ -328,10 +328,23 @@ function Set-TestAgentConfiguration
         }
     }    
 
+    $configLogFile = Join-Path $env:temp "testagentconfig.log"
+    if(Test-path -Path $configLogFile) 
```

We don't need to test the path if we are Silently Continuing.